src/app/components/todo-item/todo-item.component.ts

```diff
@@ -12,11 +14,23 @@ export class TodoItemComponent {
   @Output() checkedTodo = new EventEmitter<ITodo>();
   @Output() deletedTodo = new EventEmitter<ITodo>();
 
+  constructor(public dialog: MatDialog) {}
```

Comment:
do we really want to add dep. here ?
maybe its better to create one more output and move creation dialog logic to parent component ?